MaxMind.GeoIP2.UnitTests/ResponseHelper.cs

```diff
@@ -153,7 +154,13 @@ internal class ResponseHelper
             },
             {
                 "registered_country",
-                new JObject {{"geoname_id", 2}, {"iso_code", "CA"}, {"names", new JObject {{"en", "Canada"}}}}
+                new JObject
+                {
+                    {"geoname_id", 2},
+                    {"is_in_european_union", true },
+                    {"iso_code", "DE"},
+                    {"names", new JObject {{"en", "Germany"}}}
+                }
             },
             {
                 "represented_country",
```

Comment:
Would it be worth while setting the flag on represented_country too? We could test that as well. Looks like there are a couple instances here with GB in represented_country.